### ISSUE TITLE
Status reporter crashes on failed connection

### ISSUE THREAD

**Issue by Vortexdata** [Teamspeak-Query-Plugin-Framework/tsq-plugin-framework#87]
**Description**
If the domain of status reporter can not be resolved, an unhandled crash occurs.

**Expected behavior**
The framework should skip the status report and disable it.

**Logs or Screenshots**

```
java.net.UnknownHostException: tsqpf.vortexdata.net
        at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
        at java.net.PlainSocketImpl.connect(Unknown Source)
        at java.net.SocksSocketImpl.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at sun.security.ssl.SSLSocketImpl.connect(Unknown Source)
        at sun.security.ssl.BaseSSLSocketImpl.connect(Unknown Source)
        at sun.net.NetworkClient.doConnect(Unknown Source)
        at sun.net.www.http.HttpClient.openServer(Unknown Source)
        at sun.net.www.http.HttpClient.openServer(Unknown Source)
        at sun.net.www.protocol.https.HttpsClient.<init>(Unknown Source)
        at sun.net.www.protocol.https.HttpsClient.New(Unknown Source)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(Unknown Source)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(Unknown Source)
        at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(Unknown Source)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getOutputStream(Unknown Source)
        at net.vortexdata.tsqpf.modules.statusreporter.StatusReporter.sendData(StatusReporter.java:39)
        at net.vortexdata.tsqpf.modules.statusreporter.StatusReporter.logEvent(StatusReporter.java:64)
        at net.vortexdata.tsqpf.plugins.PluginManager.enableAll(PluginManager.java:84)
        at net.vortexdata.tsqpf.Framework.wakeup(Framework.java:229)
        at net.vortexdata.tsqpf.FrameworkContainer$1.onConnect(FrameworkContainer.java:164)
        at com.github.theholywaffle.teamspeak3.api.reconnect.ReconnectingConnectionHandler.onConnect(ReconnectingConnectionHandler.java:56)
        at com.github.theholywaffle.teamspeak3.TS3Query.connect(TS3Query.java:131)
        at com.github.theholywaffle.teamspeak3.api.reconnect.ReconnectingConnectionHandler.onDisconnect(ReconnectingConnectionHandler.java:81)
        at com.github.theholywaffle.teamspeak3.TS3Query.handleDisconnect(TS3Query.java:255)
        at com.github.theholywaffle.teamspeak3.TS3Query.lambda$submitUserTask$0(TS3Query.java:232)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
```

**Comment by TAXSET** [comment]
The issue is being worked on in the [v2.0-statusreporter-crashhandling](https://github.com/Teamspeak-Query-Plugin-Framework/tsq-plugin-framework/tree/v2.0-statusreporter-crashhandling) branch.

**Comment by TAXSET** [comment]
Fixed with pull #89.
